subsys/bluetooth/host/conn_internal.h

```diff
@@ -14,17 +14,47 @@
 typedef enum __packed {
 	BT_CONN_DISCONNECTED,         /* Disconnected, conn is completely down */
 	BT_CONN_DISCONNECT_COMPLETE,  /* Received disconn comp event, transition to DISCONNECTED */
-	BT_CONN_CONNECTING_SCAN,      /* Central passive scanning */
-	BT_CONN_CONNECTING_AUTO,      /* Central connection establishment w/ filter */
-	BT_CONN_CONNECTING_ADV,       /* Peripheral connectable advertising */
-	BT_CONN_CONNECTING_DIR_ADV,   /* Peripheral directed advertising */
-	BT_CONN_CONNECTING,           /* Central connection establishment */
+
+	BT_CONN_CENTRAL_CONNECTING,   /* Central connection establishment */
+	/** Central scans for a device preceding establishing a connection to it.
+	 *
+	 * This can happen when:
+	 * - The application has explicitly configured the stack to connect to the device,
```

Comment:
can you reflow comments to 80?

Reply:
There are already other lines here that are longer than 80 characters :) (for example for `BT_CONN_DISCONNECT_COMPLETE`. I'm not sure if it then makes sense

Reply:
Just easier to read, I know y'all in dragoon are used to 200+.
It's a personal pref so I won't block on it :+1: